ynr/apps/sopn_parsing/helpers/extract_pages.py

```diff
@@ -19,6 +20,24 @@ def extract_pages_for_ballot(ballot, manual_upload=False):
     :type ballot: candidates.models.Ballot
 
     """
+    if ballot.sopn.uploaded_file.path.endswith(".pdf"):
+        pass
+    elif ballot.sopn.uploaded_file.path.endswith(".docx"):
+        # import pdb; pdb.set_trace()
+        print(
+            f"Warning! docx file types are not supported; trying to convert {ballot.sopn.uploaded_file.path} to pdf"
+        )
+        convert(ballot.sopn.uploaded_file.path)
```

Comment:
I'm not having any luck with `doxc2pdf`. It seems to start the converting process and stalls at 0%. I've checked that the docx can be opened and has content (usig sopn `local.havant.barncroft.2021-05-06`). I will look for alternatives.

Reply:
`(Pdb++) docx_file = (f"{ballot.sopn.uploaded_file.path}") 
(Pdb++) docx_file
'/Users/virginiadooley/Documents/development/yournextrepresentative/ynr/media/official_documents/local.havant.barncroft.2021-05-06/local.havant.barncroft.2021-05-06_BY01Wkc.docx'
(Pdb++) convert(docx_file)
  0%|                                  | 0/1 [00:00<?, ?it/s]  0%|                                  | 0/1 [00:01<?, ?it/s]`